### DISCUSSION TITLE
How to exclude FLEX as an XCFramework from App Store build

### DISCUSSION THREAD

**Discussion by Jerjuan-CGI** [Flipboard/FLEX#511] [Q&A]
Hi, I have a project that contains a single App Target.  The project is presently managing dependent frameworks as XCFrameworks via Carthage.  

When I submit my app for approval to the AppStore, it is rejected because FLEX is being included in the Release build and is rejected because it contains:

ITMS-90338: Non-public API usage - The app references non-public selectors in Frameworks/FLEX.framework/FLEX: _canPerformAction:forCell:sender:, _performAction:forCell:sender:, _tableView. 

How can I include Flex as an XCFramework in Debug builds but not in the Release build for a project with a single app target?

**Comment by NSExceptional** [comment]
Are you using Cocoapods? The readme has instructions for this if you are

**Reply by Jerjuan-CGI** [reply]
No, I'm using Carthage.

**Reply by NSExceptional** [reply]
Then I do not know 😅 I'm sorry

**Reply by Jerjuan-CGI** [reply]
No worries.  I believe I found a solution

**Reply by NSExceptional** [reply]
Could you share it here so I can add it to the readme?

**Reply by NSExceptional** [reply]
@Jerjuan-CGI ?
